docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -236,6 +236,7 @@ The `--set` CLI method is more appropriate for quick test deployments.
     region: us-west-2                               # AWS region
     backendTable: teleport-helm-backend             # DynamoDB table to use for the Teleport backend
     auditLogTable: teleport-helm-events             # DynamoDB table to use for the Teleport audit log (must be different to the backend table)
+    auditLogMirrorToStdout: false                   # Whether to mirror audit log entries to stdout in JSON format (useful for external log collectors)
```

Comment:
@webvictim any reason we can't make this a default and remove the knob?

Reply:
I figured it could be a little dangerous as it would dump all audit events with no RBAC gating at all. Maybe people wouldn’t want their full audit log visible to anyone who can view the pod logs? Not sure.

Default to secure, etc.